Agoric-CLI.md

```diff
@@ -0,0 +1,87 @@
+## Agoric CLI
+
+You use the Agoric CLI to install dependencies, initialize, deploy, and start Agoric projects. `agoric` takes one of five commands as an argument. **Note**: Required argument names need not be given, only their values. Optional argument names must be given, along with their values.
+
+In general, you want to issue Agoric CLI commands in this order:
+1. `agoric init`
+2. `agoric install`
+3. `agoric start` (Usually with `--reset`)
+4. `agoric deploy`
+
+`agoric init`
+- **Required Arguments**:
+    - `project`: Name of the new project to initialize.
+- **Optional Arguments**:
+    - `--dapp-template <name>`: Use the template specified by `<name>`. Defaults to the demo DApp `dapp-simple-exchange`.
+    - `--dapp-base <base-url>`: Look under this directory for the template. Defaults to `git://github.com/Agoric/`
+- **Function**: Creates a new DApp directory named `<project>` with contents copied from the `dapp-template` argument template.
+- **Examples**:
+  - Creates a directory named `demo` with contents copied from the default `dapp-template` value `dapp-simple-exchange`.
+	- `agoric init demo`
+  - Creates a directory named `my-first-contract` with contents copied from the default `dapp-template` value `dapp-simple-exchange`.
+	  - `agoric init my-first-contract`
+  - Creates a directory named `my-contract` with contents copied from a template named `dapp-skeleton` located under the URL `file:///home/contracts`
+	  - `agoric init my-contract --dapp-template dapp-skeleton --dapp-base file:///home/contracts` 
+
+`agoric install`
+- **Required Arguments**
+  - None.
+- **Optional Arguments**
+  - None.
+- **Function**
+  - Install DApp JavaScript dependencies. This may take a while. **tyg todo**: Are these general dependencies for all Agoric projects, or specific ones connected to the directory created by `agoric init`?
+- **Examples**
+  - Installs DApp JavaScript dependencies
+	- `agoric install`
+
+`agoric start`
+- **Required Arguments**
+  - `[profile]`
+	- **tyg todo Not sure what goes here** ?
+  - `[args...]`
+    - **tyg todo Not sure what goes here?**	
+- **Optional Arguments**
+	-`--reset:`  Clear all VM state before starting.
+    - `--no-restart`: Do not actually start the VM. **tyg todo: Why
```

Comment:
I don't know about `agoric start` specifically, but sometimes this is useful to see complaints about the parsing of the arguments, or to place something in your command history without executing it.